docs/source/manual/statements/prepared/index.md

```diff
@@ -15,7 +15,7 @@ session.execute(prepared.bind("987274", "Keyboard"));
 When you prepare the statement, Cassandra will parse the query string, cache the result and return a unique identifier
 (the `PreparedStatement` object keeps an internal reference to that identifier):
 
-```ditaa
+```console
```

Comment:
Here as well, better to add ditta style, or at least alias ditta to console

Reply:
Fixed in the latest commit